koku/koku/settings.py

```diff
@@ -34,6 +34,8 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+# Version info
+GIT_COMMIT = ENVIRONMENT.get_value("GIT_COMMIT", default="local-dev")
```

Comment:
This environment variable is set during Docker build in `build_deploy.sh`. The status endpoints were simplified to use this `settings.GIT_COMMIT` instead of using a subprocess to run `git rev-parse --short HEAD`. Doing this allows us to avoid copying the `.git` folder into our final image build.